habitat-lab/habitat/datasets/rearrange/samplers/receptacle.py

```diff
@@ -232,6 +247,13 @@ def get_global_transform(self, sim: habitat_sim.Simulator) -> mn.Matrix4:
         # base class implements getting transform from attached objects
         return super().get_global_transform
 
+    def get_local_surface_center(
```

Comment:
in the future please document functions like this - one comment is enough, explain how/when we're supposed to use it

Reply:
Sure, will do that in future. For this function, I will add it [here](https://github.com/facebookresearch/habitat-lab/pull/1067) (the other PR)